packages/react-core/src/components/Card/Card.tsx

```diff
@@ -82,9 +91,12 @@ export const Card: React.FunctionComponent<CardProps> = ({
         className={css(
           styles.card,
           isHoverable && styles.modifiers.hoverable,
+          isHoverableRaised && styles.modifiers.hoverableRaised,
           isCompact && styles.modifiers.compact,
           isSelectable && styles.modifiers.selectable,
```

Comment:
I think it could be true that if someone sets `isSelectedableRaised` it sets `isSelectable` as well. So people dont NEED both props and since they are definitely related?

Reply:
@nicolethoen I'm assuming you mean that if `isSelectedRaised` is set to true `isSelectableRaised` should also be set to true automatically?

If so, I would think that if a consumer is going to set a card as `isSelectedRaised` they would also want to set `isSelectableRaised` separately so I'm not *sure* if explicitly tying them together would improve usability, I could definitely be misunderstanding/seeing things wrong though.

I could definitely see the `isSelectedRaised && is SelectableRaised` check to apply the `selected-raised` as being unneeded though, I was just copying the `isSelected` and `isSelectable` usage.

Reply:
I actually meant that if `isSelectedRaised` is true then `isSelectable` would definitely need to be true, right?